google/cloud/datastore_admin_v1/services/datastore_admin/transports/grpc_asyncio.py

```diff
@@ -22,12 +22,12 @@
 from google.auth import credentials as ga_credentials  # type: ignore
 from google.auth.transport.grpc import SslCredentials  # type: ignore
 
-import grpc
-from grpc.experimental import aio
+import grpc  # type: ignore
+from grpc.experimental import aio  # type: ignore
```

Comment:
`# type: ignore` will be removed once https://github.com/googleapis/gapic-generator-python/pull/1132/files is merged.